app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,49 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: [:edit, :update]
+  before_action :valid_user, only: [:edit, :update]
+  before_action :check_expiration, only: [:edit, :update]
```

Comment:
change code of line number 2 -4 to one line
` before_action :get_user, :valid_user, :check_expiration, only: [:edit, :update]`

Reply:
done